src/wasm-lib/kcl/src/std/shapes.rs

```diff
@@ -66,28 +78,71 @@ async fn inner_circle(
         SketchSurfaceOrGroup::SketchSurface(surface) => surface,
         SketchSurfaceOrGroup::SketchGroup(group) => group.on,
     };
-    let mut sketch_group = crate::std::sketch::inner_start_profile_at(
-        [center[0] + radius, center[1]],
+    let sketch_group = crate::std::sketch::inner_start_profile_at(
+        [data.center[0] + data.radius, data.center[1]],
         sketch_surface,
         None,
         exec_state,
         args.clone(),
     )
     .await?;
 
-    // Call arc.
-    sketch_group = crate::std::sketch::inner_arc(
-        crate::std::sketch::ArcData::AnglesAndRadius {
-            angle_start: 0.0,
-            angle_end: 360.0,
-            radius,
```

Comment:
inner_arc could not be used because under the hood it's using straight line types "toPoint", that's going to have to change but I was worried about making a possible breaking when I'm just trying to get circle in.